src/core/include/openvino/pass/sdpa_to_paged_attention.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "openvino/pass/pass.hpp"
+
+namespace ov {
+namespace pass {
+/**
+ * @brief The transformation replaces KV-cache processing part in LLMs by PagedAttention operation.
+ * \ingroup ov_pass_cpp_api
+ */
+class OPENVINO_API SDPAToPagedAttention : public ModelPass {
+public:
+    OPENVINO_RTTI("SDPAToPagedAttention");
+
+    SDPAToPagedAttention();
```

Comment:
There is no implementation. Fails when library is trying to load.

Reply:
should it be =default(); ?